src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -78,6 +79,7 @@ public class ExtensionsRunner {
     /**
      * This field is initialized by a call from {@link ExtensionsInitRequestHandler}.
      */
+    @Nullable
```

Comment:
I'm on the fence over whether to use this annotation here.  This field _can_ be null until it's initialized but we don't want it to be.  So it's a good warning that we need to null-check this if we ever read it, but that should primarily be localized to this class.  Interested in other reviewers' opinions here.

I don't think it hurts to have it, I just think it's unnecessary.

Reply:
Let's remove this @mloufra!